ProjectSettings/EditorBuildSettings.asset

```diff
@@ -0,0 +1,10 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!1045 &1
+EditorBuildSettings:
+  m_ObjectHideFlags: 0
+  serializedVersion: 2
+  m_Scenes:
+  - enabled: 1
+    path: Assets/MixedRealityToolkit-Examples/Boundary/Scenes/BoundaryTest.unity
```

Comment:
No scenes should be included in the editor build settings.

Reply:
Reset .Asset files